Tools/LocomotionTool/Scripts/BlinkVisuals.cs

```diff
@@ -180,9 +175,9 @@ void Update()
 				DrawMotionSpheres();
 
 				m_RoomScaleTransform.position = MathUtilsExt.SmoothDamp(m_RoomScaleLazyPosition, m_LocatorRoot.position,
-					ref m_MovementVelocityDelta, 0.2625f, 100f * viewerScale, Time.deltaTime);
+					ref m_MovementVelocityDelta, 0.2625f, 100f * this.GetViewerScale(), Time.deltaTime);
 
-				// Since the room scale visuals are parented under the locator root it is necessary to cache the position each frame before the locator root gets updated
+				// Since the room scale visuals are parented under the locater root it is necessary to cache the position each frame before the locater root gets updated
```

Comment:
Visual Assist was showing it as a typo, which was distracting.  I'm not actually sure if one is more/less correct.